### PR TITLE
Add the german language

### PR DESCRIPTION
**Pull request by DedBash**
We have translated the README into German so that German can understand the whole thing better we have tried to stay as close as possible to the orginal but have also observed the German grammar and below we have put the disclaimer once in English but also translated there once and of course also given us credit

### CODE REVIEW

**Review by PencilNavigator — CHANGES_REQUESTED** [review]
Duplicated.

File: translations/README_de_DE.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Inhaltsübersicht
+
+- [FAQ](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ)
+  - [Was ist das Atlas-Projekt?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#11-what-is-the-atlas-project)
+  - [Wie kann ich Atlas installieren?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#12-how-do-i-install-atlas-os)
+  - [Was wird in Atlas entfernt?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#13-whats-removed-in-atlas-os)
+- <a href="#windows-vs-atlas">Windows vs. Atlas</a>
+- [Post Install](https://github.com/Atlas-OS/Atlas/wiki/3.-Post-Install)
+- [Branding-Kit](https://github.com/Atlas-OS/Atlas/blob/main/img/brand-kit.zip?raw=true)
```
Comment (PencilNavigator) [inline translations/README_de_DE.md:32]: there is a "-" in Branding kit.
"kit" should be uncaptialized.

Comment (DedBash) [reply translations/README_de_DE.md:32]: I can do that but in german you write "Branding-Kit" or I could change it to "Marken-Kit" but it is more common to write "Branding-Kit" in german.

File: translations/README_de_DE.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Wir haben auch einen [eigenen Bereich im Discussions-Tab](https://github.com/Atlas-OS/Atlas/discussions/categories/community-artwork), damit Sie Ihre Kreationen mit anderen kreativen Genies teilen und sich vielleicht sogar inspirieren lassen können!
+
+## Disclaimer (Haftungsausschluss)
+
+By downloading, modifying, or utilizing any of these images, you agree to [Microsoft's Terms.](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm) None of these images are pre-activated, you **must** use a genuine key.
+
+### Übersetzung Haftungsausschluss
+
+Durch das Herunterladen, Ändern oder Verwenden eines dieser Installationsmedien erklären Sie sich mit [Microsofts Bedingungen](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm) einverstanden. Keines dieser Installationsmedien ist voraktiviert, Sie **müssen** einen echten Lizenzschlüssel verwenden.
```
[the remark this answers is not in the document]
Comment (DedBash) [reply translations/README_de_DE.md]: I have removed the translation of the disclaimer

File: translations/README_de_DE.md (hunk 1 above)
Comment (PencilNavigator) [reply translations/README_de_DE.md:32]: in that case, just keep it as Branding-Kit.

**Review by PencilNavigator — APPROVED** [review]
Rough check with DeepL Translate and it looks good.

**Comment by Xyueta** [comment]
cool

**Pull request merged**